chromium_edits/129.0.6658.0/chrome/installer/util/shell_util.cc.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/chrome/installer/util/shell_util.cc b/chrome/installer/util/shell_util.cc
+index 183bf96be3741..116c5b838c8c5 100644
+--- a/chrome/installer/util/shell_util.cc
++++ b/chrome/installer/util/shell_util.cc
+@@ -1526,11 +1526,12 @@ const wchar_t* ShellUtil::kDefaultFileAssociations[] = {
+ const wchar_t* ShellUtil::kPotentialFileAssociations[] = {
+     L".htm", L".html",  L".pdf",  L".shtml", L".svg",
+     L".xht", L".xhtml", L".webp", nullptr};
+-const wchar_t* ShellUtil::kBrowserProtocolAssociations[] = {L"http", L"https",
+-                                                            nullptr};
++const wchar_t* ShellUtil::kBrowserProtocolAssociations[] = {
++    L"http", L"https", L"ipfs", L"ipns", nullptr};
```

Comment:
This file is not OS-specific, but it seems like Windows keeps a bit closer to it.